### PR TITLE
EVG-17974: Fix test table e2e test and run codegen

### PR DESCRIPTION
**Pull request by julianedwards**
EVG-17974

### Description
Fix the order of expected test results for the test table e2e test after the changes made for EVG-17974—the new local test results service has a different starting point ordering of results than the previous DB test results and since we use stable sort this affects sorting by `status`.

I also ran codegen with the updated graphql schemas from the evergreen changes.

### Screenshots
N/A

### Testing
Ran this [patch build](https://spruce.mongodb.com/version/63fd2a2f57e85ab99068e67a/tasks?page=0&sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC) after updating the Evergreen module with the changes made in https://github.com/evergreen-ci/evergreen/pull/6269.

### Evergreen PR
https://github.com/evergreen-ci/evergreen/pull/6269

### CODE REVIEW

**Review by khelif96 — APPROVED** [review]
LGTM can merge after the evergreen pr is merged

**Comment by julianedwards** [comment]
Updated [patch](https://spruce.mongodb.com/version/6407a0efd1fe0704a00eb4a7/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC).

**Pull request merged**
